microsoft/devhome
# Update StyleCop

## Summary of the pull request

Updated StyleCop to the latest version to improve compatibility with latest .NET/C# feature.

## References and relevant issues

https://github.com/microsoft/devhome/issues/1259

## Detailed description of the pull request / Additional comments

- Addressed StyleCop violations after the update
- ~Added file header to `.editorconfig` to automatically add it to new `.cs` files.~
- Removed obsolete StyleCop suppressions in `GlobalSuppressions.cs`
- Added some new StyleCop suppression in `GlobalSuppressions.cs` (nice to have but with high impact in changes)

## Validation steps performed

- Changes are mostly spaces and new lines and shouldn't have any impact.
- Verified that DevHome build and run as expected.

## PR checklist
- [x] Closes #1259
